retrieval/scrape.go

```diff
@@ -74,12 +74,29 @@ var (
 		},
 		[]string{interval},
 	)
+	targetSyncIntervalLength = prometheus.NewSummary(
```

Comment:
Given how often it happens (likely no more than once per scrape interval), a histogram wouldn't gain that much.